src/App/Pages/Settings/SettingsPage/SettingsPageViewModel.cs

```diff
@@ -374,7 +381,13 @@ public async Task ApproveLoginRequestsAsync()
 
         public async Task VaultTimeoutActionAsync()
         {
-            var options = _vaultTimeoutActions.Select(o =>
+            if (_vaultTimeoutPolicy != null &&
+                !string.IsNullOrEmpty(_policyService.GetPolicyString(_vaultTimeoutPolicy, PolicyService.TIMEOUT_POLICY_ACTION)))
+            {
+                // do nothing if we have a policy set
+                return;
+            }
```

Comment:
♻️ Not to do anything here but as part of #2443 refactor, we could add a method to `Policy` called `Has(string key)` so then we could simplify the if to:
`if (_vaultTimeoutPolicy != null && _vaultTimeoutPolicy.Has(PolicyService.TIMEOUT_POLICY_ACTION))`
or
`if (_vaultTimeoutPolicy?.Has(PolicyService.TIMEOUT_POLICY_ACTION) == false)`

Reply:
I think thats a great idea! 👍